KitePrintSDK/src/main/java/ly/kite/print/PrintOrder.java

```diff
@@ -623,63 +622,6 @@ public void clearPromoCode() {
         this.promoCodeDiscount = null;
     }
 
-    /*
-     * PrintOrder history persisting methods
-     */
-
-    public void saveToHistory(Context c) {
-        List<PrintOrder> currentOrders = getPrintOrderHistory(c);
-        if (!isSavedInHistory()) {
-            storageIdentifier = getNextStorageIdentifier(currentOrders);
-        }
-
-        ArrayList<PrintOrder> updatedOrders = new ArrayList<PrintOrder>();
-        updatedOrders.add(this);
-        for (PrintOrder order : currentOrders) {
-            if (order.storageIdentifier != storageIdentifier) {
-                updatedOrders.add(order);
-            }
-        }
-
-        persistOrdersToDisk(c, updatedOrders);
-    }
-
-    public void deleteFromHistory(Context c) {
-        if (!isSavedInHistory()) {
-            return;
-        }
-
-        List<PrintOrder> orders = getPrintOrderHistory(c);
-        Iterator<PrintOrder> iter = orders.iterator();
-        while (iter.hasNext()) {
-            PrintOrder o = iter.next();
-            if (o.storageIdentifier == storageIdentifier) {
-                iter.remove();
-                break;
-            }
-        }
-
-        persistOrdersToDisk(c, orders);
-    }
-
-    private void persistOrdersToDisk(Context c, List<PrintOrder> orders) {
-        ObjectOutputStream os = null;
-        try {
-            os = new ObjectOutputStream(new BufferedOutputStream(c.openFileOutput(PERSISTED_PRINT_ORDERS_FILENAME, Context.MODE_PRIVATE)));
-            os.writeObject(orders);
-        } catch (Exception ex) {
-            // ignore, we'll just lose this order from the history now
-        } finally {
-            try {
-                os.close();
-            } catch (Exception ex) {/* ignore */}
-        }
-    }
-
-    public boolean isSavedInHistory() {
-        return storageIdentifier != NOT_PERSITED;
-    }
-
     public static int getNextStorageIdentifier(List<PrintOrder> orders) {
```

Comment:
This can go too as it's was only used by the serialization code and I imagine we'll have little use for it when (if) we do approach order history persistence correctly.